### ISSUE TITLE
OSX mouse "hotspot" in .cur file not being updated

### ISSUE THREAD

**Issue by vectorcharlie** [panda3d/panda3d#845]
When `.setCursorFilename("cursorFile.cur")` is assigned, the value of the "hotspot" on Windows is carried properly, but in OSX is not updated, leaving the "hotspot" in the top right position.

Have not tested it in Linux.

```python
wp = WindowProperties()
wp.setCursorFilename("crossHair.cur")
base.win.requestProperties(wp)
```

added .cur file with modified hotspot in zip file.

[crossHair.zip](https://github.com/panda3d/panda3d/files/4056632/crossHair.zip)

**Comment by treamology** [comment]
Looks like this was left as a TODO. Should be pretty easy to implement, I'll look into it.

**Comment by vectorcharlie** [comment]
Just build a new version on Mac 10.14 and Python 3.7 and "setCursorFilename" do not change the cursor. Windows version works fine. Using the latest Panda3d version 1.11.0

Maybe has something to do with this fix? Version 1.10.5 works fine but without the offset.

**Comment by treamology** [comment]
Looking into this, it appears your working directory must be set to wherever you are executing the script from in order for the cursor to load. This behavior is consistent between before and after this fix, so it doesn't have anything to do with it. Still, it's sort of unintuitive, so I'll look into it.
